snippets/findify-filters.liquid

```diff
@@ -6,7 +6,7 @@
 
 <div class="findify-filters-container">
   <button id="{{ close_filters_element_id }}" class="findify-modal-header">
-    <span class="findify-modal-header-title">{{ 'findify.filters.filters_btn' | t }}</span>
+    <span class="findify-modal-header-title">Filters</span>
```

Comment:
Should be `{{ 'findify.filters.filters_btn' | t }}` instead of `Filters` due to translation feature

Reply:
translation added